crates/oxc_semantic/src/scope/builder.rs

```diff
@@ -73,10 +73,8 @@ impl ScopeBuilder {
                 let scope = &self.scopes[scope];
                 if let Some(symbol_id) = scope.get().get_variable_symbol_id(&variable) {
                     // We have resolved this reference.
-                    let symbol = &mut symbol_table[symbol_id];
-                    symbol.add_references(&reference);
                     for r in reference {
```

Comment:
If the underlying storage is a Vec now, does it make sense to do this in a batch? We may get a tiny perf win.

Reply:
Fixed.